fdbserver/storageserver.actor.cpp

```diff
@@ -1340,6 +1379,8 @@ ACTOR Future<Void> getValueQ(StorageServer* data, GetValueRequest req) {
 		} else if (!i || !i->isClearTo() || i->getEndKey() <= req.key) {
 			path = 2;
 			Optional<Value> vv = wait(data->storage.readValue(req.key, IKeyValueStore::ReadType::NORMAL, req.debugID));
+			data->counters.kvBytesGet += vv.expectedSize();
```

Comment:
The name convention is 
kvGetBytes to indicate the amount of data (bytes) of gets;
kvGets to indicate the number of gets.

Reply:
Done.